### PR TITLE
requirements: Per recommendation, constrain version to <0.15

### PR DESCRIPTION
**Pull request by jcfr**
> This is the new (0.15+) interface for ``ruamel.yaml``, it is still in
> the process of being fleshed out. Please pin your dependency to
> ruamel.yaml<0.15 for production software.

Source: http://yaml.readthedocs.io/en/latest/basicuse.html

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
     [
         'numpy',
         'h5py',
-        'ruamel.yaml',
+        'ruamel.yaml<15',
```
Comment (nicain) [inline setup.py]: Just double checking that this should be 15 and not .15

**Review by nicain — COMMENTED** [review]
I really like using -e and -U before pip freeze

File: setup.py (hunk 1 above)
Comment (jcfr) [reply setup.py]: Good catch.

**Comment by jcfr** [comment]
Since all test are currently passing,  and this addresses no particular problem. I am closing it.

I originally thought this was related to #248, but it is not.

@nicain @ajtritt @oruebel I think differently, we can still re-open and integrate.

**Pull request closed without merging**